docs/Maintainer-Guidelines.md

```diff
@@ -26,7 +26,7 @@ This is all that really matters:
   [pip](https://pip.pypa.io/en/stable/).
 - Ensure that any dependencies are accurate and minimal. We don't need to
   support every possible optional feature for the software.
-- Use the GitHub squash & merge workflow where bottles aren't required.
+- Use the GitHub squash & merge or rebase & merge workflows where bottles aren't required.
```

Comment:
If it's still not clear, let's fix it. I think what you suggested works well—gives just the right amount of information while still being readable.

How about

```suggestion
- When bottles aren't required or affected, use the GitHub squash & merge workflow for a single-formula PR or rebase & merge workflow for a multiple-formulae PR. See [below](#merge-without-bottles-table) for more details.
```

I added the second `workflow` simply because I like the was it reads but I don't feel strongly about that.

I also added a link to the table to make it easy for someone who looks at that and is a little confused to get some more information.

Thought?

Reply:
I was on the fence about the second "workflow" but I'm good with it (and the link to the table). Looks good to me 👍